src/main/java/seedu/address/ui/PersonViewCard.java

```diff
@@ -0,0 +1,85 @@
+package seedu.address.ui;
+
+import java.util.Comparator;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import seedu.address.model.person.Person;
+import seedu.address.model.schedule.Schedule;
+
+/**
+ * An UI component that displays information of a {@code Person} in the viewed version.
+ */
+public class PersonViewCard extends UiPart<Region> {
+
+    private static final String FXML = "PersonViewCard.fxml";
+
+    /**
+     * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
+     * As a consequence, UI elements' variable names cannot be set to such keywords
+     * or an exception will be thrown by JavaFX during runtime.
+     *
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
+     */
```

Comment:
We should take note of this. But I don't think it should be here, we should remove it.